packages/terra-table/src/SingleSelectableRows.jsx

```diff
@@ -20,6 +20,17 @@ const defaultProps = {
   onChange: undefined,
 };
 
+const screenReaderUpdateOnSelected = (liveRegion, intl) => {
+  const liveElement = liveRegion.current;
+  liveElement.innerText = '';
+
+  // The isSelected value here actually refers to the previous state of the row since
+  // isSelected is not actually geting updated inside this listener
```

Comment:
This comment should be updated, `isSelected` isn't used here.

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/dad5154e30b000e06bf7aac427a00df44c441bd8